packages/@okta/vuepress-site/docs/api/resources/inline-hooks/index.md

```diff
@@ -542,11 +660,11 @@ curl -v -X POST \
 | Property       | Description                                                                                         | DataType                            | Nullable   | Unique   | ReadOnly   | Validation                                        |
 | -------------- | --------------------------------------------------------------------------------------------------- | ----------------------------------- | ---------- | -------- | ---------- | ------------------------------------------------- |
 | id             | Unique key for the Inline Hook.                                                                     | String                              | FALSE      | TRUE     | TRUE       | Assigned                                          |
-| status         | Status of the Inline Hook. `INACTIVE` will block execution.                                         | String                              | FALSE      | FALSE    | FALSE      | Must be either `ACTIVE` or `INACTIVE`.            |
+| status         | Status of the Inline Hook. `INACTIVE` will block execution.                                         | String                              | FALSE      | FALSE    | FALSE      | Assigned. Will be either `ACTIVE` or `INACTIVE`.            |
```

Comment:
I'm not sure what "Assigned." is supposed to mean here?

Reply:
I was just following pattern used for `id` and other fields assigned by Okta (i.e. admin cannot define them). Should it be renamed?

Reply:
Sorry, you're right. I just didn't see the others in this diff view!

Reply:
@jakubvul Would this make more sense (for all of them)?

> System assigned.

Reply:
done